yt_dlp/extractor/arte.py

```diff
@@ -59,33 +46,21 @@ class ArteTVIE(ArteTVBaseIE):
         'url': 'https://www.arte.tv/de/videos/110203-006-A/zaz/',
         'only_matching': True,
     }, {
-        'note': 'age-restricted',
-        'url': 'https://www.arte.tv/de/videos/006785-000-A/the-element-of-crime/',
-        'info_dict': {
-            'id': '006785-000-A',
-            'description': 'md5:c2f94fdfefc8a280e4dab68ab96ab0ba',
-            'title': 'The Element of Crime',
-            'timestamp': 1696111200,
-            'duration': 5849,
-            'thumbnail': 'https://api-cdn.arte.tv/img/v2/image/q82dTTfyuCXupPsGxXsd7B/940x530',
-            'upload_date': '20230930',
-            'ext': 'mp4',
-        },
-    }, {
```

Comment:
Don't remove this test unless you can add another age-restricted test video

Reply:
Can you tell me why ? This test fails as the video is no longer available on the website. May be I can comment it if you really want to keep it as a reminder...

Reply:
Tests are added to test specific code paths generally. If a test no longer functions, that path is no longer tested, and the test needs to be replaced. Its left in there as a reminder to be replaced

Reply:
So can I comment it, as a reminder, to avoid breaking the tests before to be able to execute my own test on captions ?

Reply:
add `'skip': '404 Not Found'` to the test

Reply:
Works great! I put the skipped test at the end of the tests, otherwise my test was skipped.